GeoMesaQuery.md

```diff
@@ -0,0 +1,81 @@
+# GeoMesa Indexing and Querying #
+
+## The GeoMesa Index ##
```

Comment:
> note: this can be configured to be days/weeks/months/years

I was unable to locate in the code where this configuration is implemented, a pointer would be me gratefully accepted.